adoc/admin-crio-registries.adoc

```diff
@@ -9,6 +10,13 @@ in some upstream examples.
 Please refer to: https://raw.githubusercontent.com/containers/image/master/docs/containers-registries.conf.5.md
 ====
 
+[TIP]
+====
+You can create and deploy a custom `registries.conf` during the initial bootstrap of the cluster with skuba.
+Create the file `<CLUSTER_NAME>/addons/containers/registries.conf` and apply your changes there.
+This file will be rolled out during node bootstrapping.
+====
+
 Every registry-related configuration needs to be done in the link:https://github.com/toml-lang/toml[TOML] file
 `/etc/containers/registries.conf`. After any change of this file, CRI-O
 needs to be restarted.
```

Comment:
Is this "restart required" still true?

What is the process in case the registries.conf needs further changes AFTER the initial deployment?
(i.e. new mapping for application from an other registry)

Reply:
@mssola Do you have to restart CRI-O after changing the registries file?

@Martin-Weiss If you need to update the config after initial rollout, right now the only way is to modify them individually on each existing node (or wait for the node to upgrade to automatically roll out the modified config).

I personally would expect to have some way to trigger re-distribution of any config file manually at any time but that is a feature request and needs to be handled outside the scope of this docs change. Feel free to request this as a skuba feature.

Reply:
> @mssola Do you have to restart CRI-O after changing the registries file?

Not entirely sure. I'd say so, but @saschagrunert will know for sure :smile:

Reply:
You can also reload via systemd, which should send SIGHUP to CRI-O. This triggers a configuration reload.